### ISSUE TITLE
Make the number formatting consistent between inputs and other elements?

### ISSUE THREAD

**Issue by Tofel** [InjectiveLabs/injective-dex#621]
<img width="492" alt="image" src="https://user-images.githubusercontent.com/20401585/158194261-fec8d84c-75ec-45b6-b5f0-4a8af594eaf0.png">

**Comment by ThomasRalee** [comment]
I don't think we should enhance the current implementation, I believe differentiating display format vs input format is clearer.
1. Showing available balance without comma separation would make big balances hard to read
2. Adding comma separation in input fields for big balances just makes it confusing and introduce a lot of unnecessary edge cases
